benchmark/kernels/fused_moe_triton/tuning_fused_moe_triton_splitk.py

```diff
@@ -0,0 +1,455 @@
+# Adapted from https://github.com/vllm-project/vllm/blob/main/benchmarks/kernels/benchmark_moe.py
```

Comment:
Almost the same reason with `fused_moe_splitk.y` why I use indepent file with tuning splitk best config.